src/einsteinpy/tests/test_symbolic/test_vacuum_metrics.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import sympy
+
+from einsteinpy.symbolic import ChristoffelSymbols, MetricTensor, vacuum_metrics
+
+
+def test_ChristoffelSymbols():
+    symbolstr = "t r theta phi"
+    sch = vacuum_metrics.SchwarzschildMetric()
+    chl = ChristoffelSymbols.from_metric(sch)
```

Comment:
One last change, I know I myself gave this snippet. My bad!
```
def test_SchwarzschildMetric():
    symbolstr = "t r theta phi"
    sch = vacuum_metrics.SchwarzschildMetric(symbolstr)
```

The name of the test function needs to be changed, and we do not want to reply on the default argument, so we are passing a parameter.